sql/core/src/main/scala/org/apache/spark/sql/execution/SQLExecution.scala

```diff
@@ -120,93 +120,97 @@ object SQLExecution extends Logging {
       val redactedConfigs = sparkSession.sessionState.conf.redactOptions(modifiedConfigs)
 
       withSQLConfPropagated(sparkSession) {
-        withSessionTagsApplied(sparkSession) {
-          var ex: Option[Throwable] = None
-          var isExecutedPlanAvailable = false
-          val startTime = System.nanoTime()
-          val startEvent = SparkListenerSQLExecutionStart(
-            executionId = executionId,
-            rootExecutionId = Some(rootExecutionId),
-            description = desc,
-            details = callSite.longForm,
-            physicalPlanDescription = "",
-            sparkPlanInfo = SparkPlanInfo.EMPTY,
-            time = System.currentTimeMillis(),
-            modifiedConfigs = redactedConfigs,
-            jobTags = sc.getJobTags(),
-            jobGroupId = Option(sc.getLocalProperty(SparkContext.SPARK_JOB_GROUP_ID))
-          )
-          try {
-            body match {
-              case Left(e) =>
-                sc.listenerBus.post(startEvent)
+        sparkSession.artifactManager.withResources {
```

Comment:
Can we remove this in the connect server since we are doing it here?

Reply:
Yes.